scvi/core/trainers/trainer.py

```diff
@@ -173,11 +172,8 @@ def train(self, n_epochs=400, lr=1e-3, eps=0.01, params=None, **extras_kwargs):
 
         self.on_training_begin()
 
-        for self.epoch in tqdm(
-            range(n_epochs),
-            desc="training",
-            disable=not self.show_progbar,
-            file=sys.stdout,
+        for self.epoch in track(
+            range(n_epochs), description="Training...", disable=not self.show_progbar
```

Comment:
Thoughts on changing `self.show_progbar` to `self.silent`?

Reply:
done